src/couch/src/couch_debug.erl

```diff
@@ -326,6 +328,17 @@ help(print_table) ->
           - TableSpec: List of either {Value} or {Width, Align, Value}
             where Align is either left/center/right.
 
+        ---
+    ", []);
+help(print_report) ->
+    io:format("
+        print_report(Report)
+        --------------------------------
+
+        Print a report in table form.
+          - Report: List of {InfoKey, [{InfoKey, InfoVal}]} where each InfoKey is unique
```

Comment:
I think docs are no longer represent the data structure expected as an argument.